extensions/ql-vscode/test/vscode-tests/cli-integration/debugger/debugger.test.ts

```diff
@@ -150,34 +150,36 @@ describeWithCodeQL()("Debugger", () => {
     });
   });
 
-  it("should save dirty documents before launching a debug session", async () => {
-    await withDebugController(appCommands, async (controller) => {
-      const editor = await selectForQuickEval(quickEvalLibPath, 4, 15, 4, 32);
-      await editor.edit((editBuilder) => {
-        editBuilder.insert(new Position(0, 0), "/* comment */");
-      });
-      expect(editor.document.isDirty).toBe(true);
-
-      await controller.startDebuggingSelection({
-        query: quickEvalQueryPath, // The query context. This query extends the abstract class.
-      });
-      await controller.expectLaunched();
-
-      // Should have saved the dirty document.
-      expect(editor.document.isDirty).toBe(false);
-
-      await controller.expectSucceeded();
-      await controller.expectStopped();
-
-      await editor.edit((editBuilder) => {
-        editBuilder.insert(new Position(0, 0), "/* another comment */");
+  if (process.platform !== "win32") {
```

Comment:
Is this functionality broken on windows, or is it just that the test is failing? Would you be able to add a comment to clarify this?

Reply:
Still struggling to fix a test hang that only repros on CI. Just ignore this until I get CI passing.

Reply:
In what I believe is the third :facepalm: of this PR, I finally determined that the timeouts are _not_ due to the "Save As" dialog popping up on an unexpected dirty document, but rather due to Workspace Trust, which triggers a prompt because I changed the tests to copy their workspace data to a temp folder before opening. I've disabled Workspace Trust when invoking the CLI integration tests.